include/z64.h

```diff
@@ -1344,6 +1344,19 @@ typedef struct {
     /* 0x24 */ s16 unk_24;
 } struct_80034A14_arg1; // size = 0x28
 
+#define ENTRANCE_INFO_FIELD_CONTINUE_BGM_FLAG (1 << 15)
+#define ENTRANCE_INFO_FIELD_DISPLAY_TITLE_CARD_FLAG (1 << 14)
+#define ENTRANCE_INFO_FIELD_DEFAULT_END_TRANSITION_TYPE_MASK 0x3F80
```

Comment:
I think we can probably leave out `FIELD`
I know that the member accessed to get this data is called "field", but its just called that cause it needs to be one generic variable. entrance info is enough to tell you what it is, imo